hibernate-core/src/main/java/org/hibernate/engine/jdbc/ResultSetWrapperProxy.java

```diff
@@ -96,20 +119,19 @@ private Integer findColumn(String columnName) throws SQLException {
 		return columnNameCache.getIndexForColumnName( columnName, rs );
 	}
 
-	private boolean isFirstArgColumnLabel(Method method, Object[] args) {
+	private static boolean isFirstArgColumnLabel(Method method) {
 		// method name should start with either get or update
 		if ( ! ( method.getName().startsWith( "get" ) || method.getName().startsWith( "update" ) ) ) {
 			return false;
 		}
 
 		// method should have arguments, and have same number as incoming arguments
-		if ( ! ( method.getParameterCount() > 0 && args.length == method.getParameterCount() ) ) {
+		if ( ! ( method.getParameterCount() > 0 ) ) {
 			return false;
 		}
 
 		// The first argument should be a String (the column name)
-		//noinspection RedundantIfStatement
-		if ( ! ( String.class.isInstance( args[0] ) && method.getParameterTypes()[0].equals( String.class ) ) ) {
+		if ( ! method.getParameterTypes()[0].equals( String.class ) ) {
```

Comment:
Same here, this seems unrelated to your ticket.

Reply:
Same answer.